### ISSUE TITLE
this is not the final public version.  

### ISSUE THREAD

**Issue by YalekW5435** [LolHacksRule/sexy#3]
Me and another person are maintaining a more updated version of this Library.  Were currently trying to update this to 1.40 with hopes of it finally fully working with latest standards, directx9-11, and following with the popcap coding philosophy.  Right now some stuff still needs to be changed such as unifying file outputs.

We are also trying to address some major security flaws from the outdated code, and registry writes and reads.  

https://github.com/n2g7mutf8/SexyAppFramework

**Comment by rspforhp** [comment]
it is, the 1.34 u started from is a fork made by fans

**Comment by YalekW5435** [comment]
yeah but this 1.34 version was originally sealed away by a person from assembla enterprises.  The 1.40 were "trying" to make is supposed to bring in c++20/c17 to the table...+ directx9-11, etc.  The 1.34 does not support this stuff yet...

**Comment by YalekW5435** [comment]
The actual version in question that I'm referring to is 1.7 and that has been sealed away since EA's acquisition of PopCap in the first place which is just... Ridiculous.

What I'll do is probably end up rewriting this whole library to match in accordance with C++ 17 and c18. n2g7mutf8 and the other forks provided are not in par with Windows 10 and 11 standards.  

I also want to get rid of the d2d/d3d and end up using opengl and a bunch of other stuff that does not use directdraw. 

Even though Windows primarily uses that rendering method to render its graphics, it is completely proprietary and you cannot use the set of libraries provided today...only the ones from 2010.  

Anybody who wants to join this is welcome because I'm sick and tired of having to fight between people who want to control vs people who want to cooperate.
